src/templates/en/2019/chapter.html

```diff
@@ -115,7 +115,16 @@ <h1 class="title title-lg">
                 <span class="chapter-unedited">[Unedited]</span>
                 {% endif %}
             </h1>
-            <img src="/static/images/{{ year }}/{{ get_chapter_image_dir(metadata) }}/hero_lg.jpg" class="content-banner" alt="" loading="eager" />
+            <!-- Show large image for large screens and high density screens and use webp when supported -->
+            <picture>
+              <source media="(min-width: 327px)" type="image/webp" srcset="/static/images/{{ year }}/{{ get_chapter_image_dir(metadata) }}/hero_lg.webp" />
```

Comment:
Considering this is already in the `2019` directory, do we need to be dynamic about the year?

We also have several duplicate uses of `get_chapter_image_dir(metadata)`, could we not simplify this by setting a variable?

Sort of like: `{% set chapter_image_dir =  "/static/images/2019/%s % (get_chapter_image_dir(metadata))}`

And then you can simplify all of the markup below like: 
` <source media="(min-width: 327px)" type="image/jpeg" srcset="{{ chapter_image_dir }}/hero_lg.jpg" />`

What do you think? It's not important, but might be a little neater (especially considering that it's a template for every generated chapter).

Reply:
Good idea. Pushed a commit with that. Can you re-review?